service/src/main/java/uk/gov/hmcts/reform/fpl/service/summary/CaseSummaryService.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.hmcts.reform.fpl.service.summary;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+@Service
+public class CaseSummaryService {
+
+    private final List<CaseSummaryFieldsGenerator> generators;
+    private final ObjectMapper objectMapper;
+
+    @SuppressWarnings("all")
+    public CaseSummaryService(
```

Comment:
if you want I think you can replace all the individual elements of this constructor with
```java
    public CaseSummaryService(Set<CaseSummaryFieldsGenerator> generators, ObjectMapper objectMapper) {
```
see these [spring docs](https://docs.spring.io/spring-framework/docs/current/reference/html/core.html#beans-autowired-annotation) unless you prefer to be explicit

Reply:
I prefer to leave explicitly typed so It can be tested without needing to use an integration test. Moreover that feels more predictable to me.

Reply:
yeah I am fine with explicit, just giving options whilst teaching myself something